lib/developerTestAccounts.ts

```diff
@@ -58,18 +51,18 @@ export async function validateDevTestAccountUsageLimits(
   const limit = data.maxTestPortals;
   const count = data.results.length;
   if (count >= limit) {
-    const hasDevTestAccounts = getHasDevTestAccounts(accountConfig);
+    const hasDevTestAccounts = getHasDevTestAccounts(account);
     if (hasDevTestAccounts) {
       throw new Error(
         i18n('lib.developerTestAccount.create.failure.alreadyInConfig', {
-          accountName: accountConfig.name || accountId,
+          accountName: account.name,
```

Comment:
Are accounts guaranteed to have names?

Reply:
They are - in the case where there isn't a name, the account Id gets set as the name